.bacon.yml

```diff
@@ -0,0 +1,50 @@
+test_suites:
+    - name: lint
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '1'
+      timeout: '60'
```

Comment:
Let's modify this to `15`. Any lint checks that take longer than that imply a serious issue.